packages/types-core/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "@subql/types-core",
+  "version": "2.2.1-0",
```

```suggestion
  "version": "0.0.0-0",
```
Then we won't start on some random version